### ISSUE TITLE
Add Open Graph metadata

### ISSUE THREAD

**Issue by m-mohr** [radiantearth/stac-browser#77]
Add Open Graph metadata so that social media show nice visualizations of the page.

https://ogp.me/

**Comment by rowanwins** [comment]
Depending on how nice you want the links to be this may be challenging because the app is being rendered dynamically rather than with SSR. See https://stackoverflow.com/questions/46718161/how-to-do-open-graph-images-in-a-vue-js-spa for some more context.

**Comment by m-mohr** [comment]
Yes, we may not be able to easily generate an image, but we can at least provide some additional metadata.

For the image, we may just provide nice imagery later with an extension that generates them. This is a similar issue as with validation and tiles where we use external services (staclint, radiant tiler). We may want to also have a dedicated service for this.

**Comment by m-mohr** [comment]
Unless we do SSR, this is impossible to do.
